lib/chef/knife/windows_cert_generate.rb

```diff
@@ -108,11 +108,32 @@ def write_certificate_to_file cert, file_path, rsa_key
         File.open(file_path + ".b64", "wb") { |f| f.print Base64.strict_encode64(pfx.to_der) }
       end
 
+      def is_other_certificates_present?(file_path)
+        is_certs_exists = false
```

Comment:
Suggestion for method and variable names:
1. `is_other_certificates_present?` method name can be `certificates_already_exist?`
2. `is_certs_exists` variable name can be `certs_exist`